model/Human-GEM.yml

```diff
@@ -117602,15 +117658,21 @@
       - id: "MAR03481"
       - name: ""
       - metabolites: !!omap
+          - MAM01285c: 1
+          - MAM01371c: -1
+          - MAM01597c: 1
+          - MAM02039c: 1
+          - MAM02040c: -2
```

Comment:
> Edited MAR03481 to be MAM02747c + MAM01371c + 2 MAM02040m -> MAM02746x + MAM01597c + MAM02039c + MAM01285c + MAM02751c, GPR: ENSG00000117528, references: PMID:24333844;PMID:33500543

There is a discrepancy between the description in PR and changed file,  `MAM02040m` was set as reactant in PR but `MAM02040c` in the file.  May be it's necessary to check which one is correct.

Reply:
ah yea it was supposed to be `MAM02040c`; I'm not sure why I had `MAM02040m` in the issue and PR, but I've updated both of those to match what I actually wrote in `model/Human-GEM.yml`